app/services/user/index.ts

```diff
@@ -893,6 +907,16 @@ export class UserService extends PersistentStatefulService<IUserServiceState> {
       return validatePlatformResult;
     }
 
+    if (!this.sideNavService.views.hasLegacyMenu && !this.views.auth.hasRelogged) {
+      this.sideNavService.setNewUserLogin();
+    } else if (
+      this.sideNavService.views.hasLegacyMenu &&
+      this.dismissablesService.views.shouldShow(EDismissable.NewSideNav)
+    ) {
+      this.sideNavService.setLegacyView();
+    }
+
+    this.dismissablesService.dismiss(EDismissable.LoginPrompt);
```

Comment:
just caught this but i think this functionality should live in the SidenavService and subscribe to the login `Subject`. we don't really want side-effects living in this function or else it'll get really clogged up

Reply:
Ok sounds good